content/healthchecks/readinessprobe.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Configure Readiness Probe"
+chapter: true
+weight: 10
+---
+
+# Configure Readiness Probe
+
+```
+kubectl apply -f ~/environment/healthchecks/readiness-deployment.yaml
+```
+
+The above command creates a deployment with 3 replicas and readiness probe as described in ~/environment/healthchecks/readiness-deployment.yaml.
+
+
+```
+kubectl get pods -l app=readiness-deployment
+
+NAME                                    READY     STATUS    RESTARTS   AGE
```

Comment:
can you separate input from outputs so that copy/paste is easier?

Reply:
done